code/game/gamemodes/steal_items.dm

```diff
@@ -14,6 +14,8 @@
 	var/location_override
 	/// Do we have a special item we give to somewhen when they get this objective?
 	var/special_equipment = null
+	///If a steal objective has forbiden jobs, and the forbiden jobs would not be in the possession of this item, set this to false
```

Comment:
```suggestion
	/// If a steal objective has forbiden jobs, and the forbiden jobs would not be in the possession of this item, set this to false
```

Reply:
Misclick. Pretend this was a requested change.

Reply:
already done